nucleus/common/internal-api/src/main/java/org/glassfish/internal/data/ApplicationInfo.java

```diff
@@ -321,7 +321,7 @@ public void load(ExtendedDeploymentContext context, ProgressTracker tracker)
         span.close();
 
         if (events!=null) {
-            span = tracing.startSpan(TraceContext.Level.APPLICATION, name, DeploymentTracing.AppStage.LOAD_EVENTS);
+            span = tracing.startSpan(TraceContext.Level.APPLICATION, name, DeploymentTracing.AppStage.PROCESS_EVENTS, Deployment.APPLICATION_LOADED.type());
             events.send(new Event<ApplicationInfo>(Deployment.APPLICATION_LOADED, this), false);
```

Comment:
Please also update the tracing snippet used in ApplicationInfo#start method:

````
DeploymentSpan innerSpan = tracing.startSpan(TraceContext.Level.APPLICATION, getName(), DeploymentTracing.AppStage.PROCESS_EVENTS, Deployment.APPLICATION_STARTED.type());
events.send(new Event<ApplicationInfo>(Deployment.APPLICATION_STARTED, this), false);
````

Reply:
Will do.